Makefile

```diff
@@ -190,6 +190,9 @@ $(GOPATH1)/bin/%:
 test: build
 	go test $(GOTAGS) -race $(UNIT_TEST_SOURCES) -timeout 3600s
 
+ci-test: build
```

Comment:
we should reference ci-build here since it handles everything that build does. Alternatively, we should change ci-build to reference build instead of "buildsrc gen" which is what the build target does

Reply:
Updated build to ci-build.